### PR TITLE
update ui

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- update some texts

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -82,7 +83,7 @@ export function getEtherscanLink(
 }
 
 export function getEtherscanLinkText(chainId: ChainId): string {
-  return NETWORKS_INFO[chainId].etherscanName
+  return t`View transaction` // NETWORKS_INFO[chainId].etherscanName
```
Comment (namgold) [inline src/utils/index.ts:86]: remove comment

Comment (namgold) [reply src/utils/index.ts:86]: no need to keep it

Comment (namgold) [reply src/utils/index.ts:86]: ah, you fix wrong place

Comment (nguyenhoaidanh) [reply src/utils/index.ts]: why it's wrong, bro?

File: src/utils/index.ts [hunk 2]
```diff
@@ -82,7 +83,7 @@ export function getEtherscanLink(
 }
 
 export function getEtherscanLinkText(chainId: ChainId): string {
-  return NETWORKS_INFO[chainId].etherscanName
+  return t`View transaction`
```
Comment (namgold) [inline src/utils/index.ts:86]: This is wrong fix. This function is `getEtherscanLinkText`, so it should return scan page name. Also it used at many other place

File: src/utils/index.ts (hunk 1 above)
Comment (nguyenhoaidanh) [reply src/utils/index.ts]: I think he want to change all place

Comment (namgold) [reply src/utils/index.ts]: image
you changed 3 place

Comment (namgold) [reply src/utils/index.ts]: I dont think AddressInputPanel is correct one

File: src/utils/index.ts (hunk 2 above)
Comment (hungdoansy) [reply src/utils/index.ts:86]: Agree with Nam. This doesn't quite look good to me. The func name is getEtherscanLink, I expect to get a link to etherscan or some sort. But it just returns View Transaction?

Comment (nguyenhoaidanh) [reply src/utils/index.ts]: I just want to fast update. okay. I removed that function

**Review by namgold — APPROVED** [review]

**Review by hungdoansy — APPROVED** [review]

**Pull request merged**